src/test/datascience/notebook/nonPythonKernels.vscode.test.ts

```diff
@@ -170,6 +170,7 @@ suite('DataScience - VSCode Notebook - Kernels (non-python-kernel) (slow)', () =
         }
     });
     test('Can run a Julia notebook', async function () {
+        this.skip(); // Flakey, tracked by issue 4453
```

Comment:
I think its better to `return this.skip()`, else rest of the code still executes, and fails silently (as test has been skipped)...

Reply:
Yeah flaky, failed on me yesterday once...

Reply:
Ahh, didn't know that. In the IDE the code after this.skip looks greyed out as unreachable, but I can change to return this.skip() if it safer. Will do.